src/StatusQ/Core/Theme/StatusLightTheme.qml

```diff
@@ -145,15 +145,19 @@ ThemePalette {
     miscColor11: getColor('brown2')
     miscColor12: getColor('green5')
 
-    accountColors: [
-        getColor('blue'),
-        getColor('brown2'),
-        getColor('violet'),
-        "#1D806F",
-        getColor('red2'),
-        getColor('green2'),
-        getColor('purple'),
-        getColor('brown')
```

Comment:
Why are we removing `accountColors`?
Those are needed for wallet account options.

Reply:
Yes, I've changed all occurrences of `accountColors` to `userCustomizationColors` from: https://www.figma.com/file/WMmmNowe0sTVCaxadUB8mE/Desktop-Foundations?node-id=1%3A609

"total number of colours in the user customisation colour set is 12, but in some contexts only a subset of these colours will be used e.g. 10 colours, or 8 colours from that 12 colour pallet, they can be used and referenced throughout the product wherever user customisation colours are required e.g. this set of colours is used for colouring Accounts in the user's wallet, and is also used in the Community create and edit screens."     John

Reply:
Sure, done.